config/locales/exceptions.en.yml

```diff
@@ -296,6 +296,12 @@ en:
         code: 'MHVACCTCREATION159'
         detail: 'User is not matched or correlated in MVI'
         status: 422
+      MHV_STATUS502:
```

Comment:
Admittedly the exception naming scheme is not super-consistent across vets-api, but the intent was more to define API-by-API errors. So in this case we should name the errors "OAUTH_XYZ" or "AUTHTOKEN_XYZ". Also the backend in question is MVI, not MHV.

Reply:
Will change the name to AUTHTOKEN_502 then